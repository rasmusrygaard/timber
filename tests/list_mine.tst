list /
list //
list ///
create /test 123
create /test/a 456
create /test/b 789
list /test
list /test/a
list /test/b
list test
remove /test/b
list /test
list /missing_key
list /
remove /test/a
list /test
list /
