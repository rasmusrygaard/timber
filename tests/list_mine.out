MALFORMED KEY
MALFORMED KEY
CREATED
CREATED
CREATED
a
b
MALFORMED KEY
REMOVED
a
KEY NOT FOUND
test
REMOVED
test
